dynomite/src/lib.rs

```diff
@@ -153,23 +177,33 @@ pub type Attributes = HashMap<String, AttributeValue>;
 /// }
 ///
 /// impl FromAttributes for Person {
-///     fn from_attrs(attrs: Attributes) -> Result<Self, AttributeError> {
+///     fn from_mut_attrs(attrs: &mut Attributes) -> Result<Self, AttributeError> {
 ///         Ok(Self {
 ///             id: attrs
-///                 .get("id")
-///                 .and_then(|val| val.s.clone())
-///                 .ok_or(AttributeError::MissingField { name: "id".into() })?,
+///                 .remove("id")
+///                 .and_then(|val| val.s)
+///                 .ok_or_else(|| AttributeError::MissingField { name: "id".into() })?,
 ///         })
 ///     }
 /// }
 ///
-/// impl Into<Attributes> for Person {
-///     fn into(self: Self) -> Attributes {
-///         let mut attrs = HashMap::new();
+/// impl IntoAttributes for Person {
+///     fn into_mut_attrs(
+///         self,
+///         attrs: &mut Attributes,
+///     ) {
 ///         attrs.insert("id".into(), "123".to_string().into_attr());
-///         attrs
 ///     }
 /// }
+///
+/// // Unfortunately `dynomite` is not able to provide a blanket impl for this trait
```

Comment:
Why introduce this when derive Attributes on a type?

Reply:
This is an example of not using derive attributes and making a manual impl:

> Below is an example of doing this manually for demonstration.